components/motor/errors.go

```diff
@@ -19,3 +19,9 @@ func NewResetZeroPositionUnsupportedError(motorName string) error {
 func NewFeatureUnsupportedError(feature Feature, motorName string) error {
 	return errors.Errorf("motor named %s must support feature motor.%s", motorName, feature)
 }
+
+// NewZeroRPMError returns an error representing a request to move a motor at
+// zero speed (i.e., moving the motor without moving the motor).
```

Comment:
nit : moving the motor at a null speed

Reply:
Not sure I agree with this phrasing: it happens when the RPM is 0, not when it's `nil`. 

...nobody's going to pass in a `nil` RPM, are they? Do we need to check for that, too?